plone/app/z3cform/templates/radio_input_single.pt

```diff
@@ -4,47 +4,24 @@
         item python:args[0];
       "
       tal:omit-tag=""
->
-  <input class=""
-         id=""
-         accesskey=""
-         alt=""
-         name=""
-         tabindex=""
-         title=""
+><!--! NOTE:
+           we cannot autoset the attributes here with "view/attributes",
+           because the "id" would get overwritten with the field_id
+           instead of the correct sequence_item_id -->
+  <!--! NOTE #2:
+           Since most people import the "RadioFieldWidget"
+           directly from z3c.form.browser.radio our adapter
+           which sets the Bootstrap "form-check-input" css class
+           isn't invoked. For this reason we manually add the
+           class here in the template.
+           When all imports are refactored to
+           "plone.app.z3cform.widgets.radio.RadioFieldWidget"
+           this can be cleaned up. -->
```

Comment:
```suggestion
><tal:c condition="nothing"
    NOTE:
    Since most people import the `RadioFieldWidget` directly from
    `z3c.form.browser.radio` our adapter which sets the Bootstrap
    `form-check-input` CSS class isn't invoked. For this reason we manually add
    the class here in the template. When all imports are refactored to
    `plone.app.z3cform.widgets.radio.RadioFieldWidget` this can be cleaned up.
</tal:c>
```